webapp/resources/templates/fragments/pwdupdateform.html

```diff
@@ -7,7 +7,12 @@
 </script>
 
 <div class="alert alert-info">
-    <h2 th:utext="#{screen.mustchangepass.heading}"/>
+    <div th:if="${execInfo.processedTemplateName} == 'casExpiredPassView'">
```

Comment:
What if the casExpiredPassView set a variable, that it would be checked (true/false) in pwdupdateform? Is that ok?